src/offers/lnd_requests.rs

```diff
@@ -613,4 +671,156 @@ mod tests {
         let hops = reply_path.blinded_hops();
         assert!(hops.len() == 2);
     }
+
+    fn get_blinded_payment_path() -> BlindedPaymentPath {
```

Comment:
This is a move from (the now deleted) `lndk_offers`, right? Is it possible to have the move in the same commit and split out of this one? If not that's fine. Just started reviewing this test function and realised it was a move :)